Client/Ecosia/Migration/LoadingScreen.swift

```diff
@@ -67,7 +67,7 @@ final class LoadingScreen: UIViewController {
             migrate()
         }
 
-        if let code = referralCode {
+        if Referrals.isEnabled, let code = referralCode {
```

Comment:
also this should stay as is. It should have said `Referrals.isActivatedForBuild` when we still had it, but that was only the case fpor 6.0.3